addon/ng2/blueprints/component/files/__path__/__name__.component.spec.ts

```diff
@@ -10,14 +10,28 @@ import {
   ComponentFixture,
   TestComponentBuilder
 } from 'angular2/testing';
-import {provide} from 'angular2/core';
+import {provide} from 'angular2/core';<% if (route) { %>
+import {Router, RouteParams} from 'angular2/router';<% } %>
 import {<%= classifiedModuleName %>Component} from './<%= dasherizedModuleName %>.component';
+<% if (route) { %>
+class MockRouter {
+  navigate() { }
```

Comment:
I think we can remove the navigate method. At the time I used it for my code, but it doesn't seem to be used here.